src/LibraryViewExtension/LibraryViewController.cs

```diff
@@ -137,6 +139,7 @@ public void CreateNode(string nodeName)
                 //Create the node of given item name
                 var cmd = new DynamoModel.CreateNodeCommand(Guid.NewGuid().ToString(), nodeName, -1, -1, true, false);
                 commandExecutive.ExecuteCommand(cmd, Guid.NewGuid().ToString(), ViewExtension.ExtensionName);
+                AddInstrumentation("Search-NodeAdded", nodeName);
```

Comment:
maybe we can pull out the magic strings and make them constants?

Reply:
sure